lib/active_model/serializer/logging.rb

```diff
@@ -0,0 +1,32 @@
+module ActiveModel
+  class Serializer
+    module Logging
+      extend ActiveSupport::Concern
+
+      included do
+        cattr_accessor(:logger) do
+          ActiveSupport::TaggedLogging.new(ActiveSupport::Logger.new(STDOUT))
+        end
+      end
+
+      class LogSubscriber < ActiveSupport::LogSubscriber
+        def render(event)
+          serializer = event.payload[:serializer]
+          adapter = event.payload[:adapter]
+          duration = event.duration.round(2)
+          logger.tagged('AMS') do
+            info do
+              "Rendered #{serializer.name} with #{adapter.class} (#{duration}ms)"
```

Comment:
The more work you can do in the `info` block the better (i.e. all the locals here), since it is only evaluated when the info log level. (I think it should be debug, so this should be configurable.. `logger.add(Logger::DEBUG) { "Rendered..." }` would be fine